tests/test_trace.py

```diff
@@ -1309,7 +1321,8 @@ def test_init_propagate_lambda_identifier_env(wrapped_init, _create):
         keys_to_ignore=None,
         keys_to_allow=None,
         split_on_send=False,
-        propagate_lambda_id=True
+        propagate_lambda_id=True,
+        add_log_id=False,
```

Comment:
let's also add an environment variable to control it, and also add it to the env vars table in the README

Reply:
:+1: